tools/ci/mkchlog-snippet.py

```diff
@@ -89,14 +89,6 @@ def as_snippet(self) -> str:
                 + " via "
             )
         item += f"{self.as_link()} (by {self.author.as_link()})"
-        item = textwrap.fill(
-            item,
-            width=79,
-            initial_indent="- ",
-            subsequent_indent="  ",
-            break_long_words=False,
-            break_on_hyphens=False,
-        )
```

Comment:
that is for now.  For the action - filed https://github.com/datalad/release-action/issues/8